test/functional/ui/popupmenu_spec.lua

```diff
@@ -1198,20 +1198,20 @@ describe('builtin popupmenu', function()
 
     command("split")
     screen:expect([[
-      xx                                      |
       choice^                                  |
+      {1:~                                       }|
       {n:word           }{1:                         }|
       {s:choice         }{4:                         }|
       {n:text           }                         |
-      {n:thing          }                         |
+      {n:thing          }{1:                         }|
       {3:[No Name] [+]                           }|
       {2:-- INSERT --}                            |
```

Comment:
This is an edge case: `:split` comman sent while the popupmenu is showing. The pum is repositioned next time the user does `<c-n>`. Doesn't seem important to redraw it on the `:split`. But I mention it anyway. @bfredl

Reply:
"Ideally" it should be resized immediately, but I agree fixing it is not top priority. As long as split caused by the completion mechanism itself (preview window) is working.